docs/content/en/docs/overview/_index.md

```diff
@@ -6,18 +6,14 @@ description: >
   Provides an overview of EKS Anywhere
 ---
 
-EKS Anywhere uses the `eksctl` executable to create a Kubernetes cluster in your environment.
-Currently it allows you to create and delete clusters in a vSphere environment.
-You can run cluster create and delete commands from an Ubuntu or Mac administrative machine.
+EKS Anywhere lets you create a Kubernetes cluster on premises to a chosen provider.
+Supported providers include Bare Metal and vSphere.
```

Comment:
Should we say "Bare Metal" is tinkerbell in the CLI? Something like

```suggestion
Supported providers include Bare Metal (via Tinkerbell) and vSphere.
```
Or do we think people will figure it out when using `--help` output?